events/comic_book_day/meta.md

```diff
@@ -0,0 +1,7 @@
+---
+start_date: August 13
+end_date: August 14
+---
+
+**Free Comic Book Day**
+[Free Comic Book Day](https://en.wikipedia.org/wiki/Free_Comic_Book_Day) is an annual promotional effort by the North American comic book industry to help bring new readers into independent comic book stores.
```

Comment:
Could you add another line here describing that it's usually on on the first Saturday in May, but has been rescheduled for Aug 14th for 2021?

This will help us figuring out what date to change it to each year.

Reply:
Added this